Scripts/modelsystem.py

```diff
@@ -279,6 +279,7 @@ def run_iteration(self, previous_iter_impedance, iteration=None):
             if tp == "aht":
                 self._update_ratios(impedance[tp], tp)
             if iteration=="last":
+                impedance[tp]["std_time"] = previous_iter_impedance[tp]["time"]
```

Comment:
This consumes a lot of hard disk space, because we store copies of travel time matrices for all modes. What if we would save uncongested transit times in `impedance[tp]["time"]["transit_uncongested"]` instead? Would then need to modify `MockAssignment` accordingly of course.